iot/api-client/beta-features/manager/README.md

```diff
@@ -20,77 +21,40 @@ mvn clean compile assembly:single
 
 ## Running the sample
 
-The following description summarizes the sample usage:
-
-    usage: DeviceRegistryExample [--cloud_region <arg>] --command <arg>
-           [--data <arg>] [--device_id <arg>] [--project_id <arg>]
-           [--registry_name <arg>]
-    Cloud IoT Core Commandline Example (Device / Registry management):
-
-        --cloud_region <arg>    GCP cloud region.
-        --command <arg>         Command to run:
-                                send-command
-        --data <arg>            The command data (string or JSON) to send to
-                                the specified device.
-        --device_id <arg>       Name for your Device.
-        --project_id <arg>      GCP cloud project name.
-        --registry_name <arg>   Name for your Device Registry.
-
-For example, if your project ID is `blue-jet-123`, your service account
-credentials are stored in your home folder in creds.json and you have generated
-your credentials using the shell script provided in the parent folder, you can
-run the sample as:
-
-    mvn exec:java \
-      -Dexec.mainClass="com.example.cloud.iot.examples.DeviceRegistryExample" \
-      -Dexec.args="-project_id=blue-jet-123 \
-                   -registry_name=your-registry-id \
-                   -device_id=your-device-id \
-                   -command=send-command \
-                   -data=hello"
-
-# Cloud IoT Core Java MQTT Command example
-
-This sample app receives commands sent by the command app.
+The following code will run the sample
 
-Note that before you can run the sample, you must configure a Google Cloud
-PubSub topic for Cloud IoT Core and register a device as described in the
-[parent README](../README.md).
-
-## Setup
+    mvn exec:java -Dexec.mainClass="com.example.cloud.iot.examples.DeviceGatewayExample"
```

Comment:
sometimes GH flavored markdown does not render without a newline above the code line, make sure it looks good in preview